src/index.html

```diff
@@ -10,8 +10,121 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
```

Comment:
Add proper indentation and put closing brackets on a newline when attributes on a newline. Fix it everywhere so it looks like that:
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
    >
```